### PR TITLE
Amenities setup

### PR DESCRIPTION
**Pull request by reckocp**
Added controller, columns and views for Amenities

### CODE REVIEW

File: app/views/amenities/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div> Amenity Show Page
+  <p>Name <%=@amenity.name %>
```
Comment (reicht) [inline app/views/amenities/show.html.erb]: Needs a closing </p>

Comment (reckocp) [inline app/views/amenities/show.html.erb]: Fixed.

**Comment by brentmartin** [comment]
Yup

**Comment by reicht** [comment]
👍

**Comment by jaredpowers** [comment]
👍

**Pull request merged**